pkgs/servers/sunshine/default.nix

```
@@ -171,6 +171,7 @@ stdenv.mkDerivation rec {
     description = "Sunshine is a Game stream host for Moonlight";
```

```suggestion
    description = "Game stream host for Moonlight";
```
See https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes